changes/ce/feat-14459.en.md

```diff
@@ -0,0 +1,15 @@
+Added support for Fallback Actions.
+
+These are triggered when a message fails to be successfully processed by a Data Integration Action, including when it's dropped due to buffer overflow or to its time to live being reached.  They can be configured for all Data Integration Actions.
+
+Sample configuration snippet for a Kafka Action:
+
+```hcl
```

Comment:
is this a supported syntax ? it may cause a warning message for emqx-docs markdown render

Reply:
Yes, this is how to specify the syntax highlighting to use.  Hocon happens to be well highlighted by [HCL](https://developer.hashicorp.com/terraform/language/syntax/configuration) and Hocon itself is not recognized by github AFAIK.